### PR TITLE
community: add PT-BR translation for remote-schemas.md

### PR DESCRIPTION
**Pull request by jrcamelo**
### Description
Added docs translation for PT-BR (Brazilian Portuguese), related to the following files:
- remote-schemas.md

Fixed typos in the following files:
- remote-schemas.md
- README.portuguese_br.md

### Changelog

- [ ] `CHANGELOG.md` is updated with user-facing content relevant to this PR. If no changelog is required, then add the `no-changelog-required` label.

### Affected components

- [ ] Server
- [ ] Console
- [ ] CLI
- [x] Docs
- [ ] Community Content
- [ ] Build System
- [ ] Tests
- [ ] Other (list it)

### Related Issues 
relates to #5851 

#### Catalog upgrade

Does this PR change Hasura Catalog version?
- [x] No
- [ ] Yes
  - [ ] Updated docs with SQL for downgrading the catalog 

#### Metadata

Does this PR add a new Metadata feature?
- [x] No
- [ ] Yes
  - Does `run_sql` auto manages the new metadata through schema diffing?
    - [ ] Yes
    - [ ] Not required
  - Does `run_sql` auto manages the definitions of metadata on renaming?
    - [ ] Yes
    - [ ] Not required
  - Does `export_metadata`/`replace_metadata` supports the new metadata added?
    - [ ] Yes
    - [ ] Not required

#### GraphQL
- [x] No new GraphQL schema is generated
- [ ] New GraphQL schema is being generated:
   - [ ] New types and typenames are correlated
   
   

#### Breaking changes

- [x] No Breaking changes
- [ ] There are breaking changes:

### CODE REVIEW

File: translations/remote-schemas.portuguese_br.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+A maneira mais rápida de testar o esquema remoto é via Heroku.
+
+1. Clique no botão a seguir para implantar o GraphQL Engine no Heroku com o add-on gratuito do Postgres:
+
+    [![Deploy to Heroku](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/hasura/graphql- motor-heroku)
+
+2. Abra o console Hasura
+
+   Visite `https://<app-name>.herokuapp.com` (*substitua \<app-name\> pelo nome do seu aplicativo*) para abrir o console de administração.
```
Comment (rafaeelaudibert) [inline translations/remote-schemas.portuguese_br.md:35]:
```suggestion
   Visite `https://<nome-aplicativo>.herokuapp.com` (*substitua \<nome-aplicativo\> pelo nome do seu aplicativo*) para abrir o console de administração.
```

File: translations/remote-schemas.portuguese_br.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+3. Combine seu primeiro esquema remoto e consulte-o
+
+   No console de administração, abra a guia ``Esquemas remotos`` e clique no botão ``Adicionar``. Preencha os seguintes detalhes:
+   * Nome do esquema remoto: ``países`` (*um apelido para este esquema remoto*).
+   * URL do servidor GraphQL: ``https://countries.trevorblades.com/``(*uma API GraphQL pública que usaremos para verificar rapidamente este recurso; mantida por [@trevorblades](https://github.com/trevorblades)*.
+   * Ignore as configurações restantes e clique no botão ``Add Remote Schema``.
+
+   Vá para a guia ``GraphiQL` e execute a seguinte consulta (*cole-o na janela de consulta à esquerda e clique no botão* ▶️ * (reproduzir)*):
```
Comment (rafaeelaudibert) [inline translations/remote-schemas.portuguese_br.md:44]: Here there is a missing closing backtick
```suggestion
   Vá para a guia ``GraphiQL`` e execute a seguinte consulta (*cole-o na janela de consulta à esquerda e clique no botão* ▶️ * (reproduzir)*):
```

File: translations/remote-schemas.portuguese_br.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+   ```
+
+   Você pode explorar os tipos GraphQL do esquema remoto usando o explorador ``Docs`` no canto superior direito da interface ``GraphiQL``.
+
+## Boilerplates
+
+Boilerplates para servidores GraphQL personalizados em linguagens/estruturas populares estão disponíveis.
+
+* [Boilerplates comuns](community/boilerplates/graphql-servers) que podem ser implantados em qualquer lugar.
+* [Boilerplates sem servidor](https://github.com/hasura/graphql-serverless) que podem ser implantados em plataformas sem servidor como AWS Lambda, etc.
```
Comment (rafaeelaudibert) [inline translations/remote-schemas.portuguese_br.md:66]: I think it is worth to keep the expression `serverless` as it is easily recognizable.
```suggestion
* [Boilerplates sem servidor (*serverless*)](https://github.com/hasura/graphql-serverless) que podem ser implantados em plataformas sem servidor como AWS Lambda, etc.
```

File: translations/remote-schemas.portuguese_br.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+Observe que padrões para mais linguagens, frameworks, plataformas sem servidor, etc. estão sendo iterados e as contribuições da comunidade são muito bem-vindas.
+
+
+## Ressalvas
+
+**Limitações atuais**:
+
+* Nomenclatura: os nomes dos tipos e dos nós precisam ser exclusivos em todos os esquemas mesclados (correspondência com distinção entre maiúsculas e minúsculas). Nas próximas iterações, o suporte para mesclar tipos com o mesmo nome e estrutura estará disponível.
```
Comment (rafaeelaudibert) [inline translations/remote-schemas.portuguese_br.md:75]:
```suggestion
* Nomenclatura: os nomes dos tipos e dos nós precisam ser exclusivos em todos os esquemas mesclados (correspondência com distinção entre maiúsculas e minúsculas). Nas próximas versões, o suporte para mesclar tipos com o mesmo nome e estrutura estará disponível.
```

File: translations/remote-schemas.portuguese_br.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+* Nós de diferentes servidores GraphQL não podem ser usados ​​na mesma consulta/mutação. Todos os nós de nível superior devem ser do mesmo servidor GraphQL.
+* As assinaturas no servidor GraphQL remoto não são suportadas.
+
+Essas limitações serão abordadas nas próximas versões.
+
+## Documentação
+
+Leia a [documentação] completa (https://hasura.io/docs/1.0/graphql/manual/remote-schemas/index.html).
```
Comment (rafaeelaudibert) [inline translations/remote-schemas.portuguese_br.md:83]: Here you linked it in the incorrect way, it should be:
```suggestion
Leia a [documentação](https://hasura.io/docs/1.0/graphql/manual/remote-schemas/index.html) completa.
```

**Review by rafaeelaudibert — CHANGES_REQUESTED** [review]
Added some comments about small changes.

Also, what do you think about keeping the original word `schema` instead of the `esquema` translation? I think it is more easily recognizable keeping in English.

File: translations/remote-schemas.portuguese_br.md (hunk 1 above)
Comment (jrcamelo) [reply translations/remote-schemas.portuguese_br.md:35]: This does look more intuitive indeed

File: translations/remote-schemas.portuguese_br.md (hunk 2 above)
Comment (jrcamelo) [reply translations/remote-schemas.portuguese_br.md:44]: Good catch

File: translations/remote-schemas.portuguese_br.md (hunk 3 above)
Comment (jrcamelo) [reply translations/remote-schemas.portuguese_br.md:66]: That makes sense

File: translations/remote-schemas.portuguese_br.md (hunk 5 above)
Comment (jrcamelo) [reply translations/remote-schemas.portuguese_br.md:83]: Shoot, missed that!

File: translations/remote-schemas.portuguese_br.md (hunk 4 above)
Comment (jrcamelo) [reply translations/remote-schemas.portuguese_br.md:75]: Alright

**Comment by jrcamelo** [comment]
@rafaeelaudibert could you take another look?

**Pull request closed without merging**